packages/tables/resources/views/index.blade.php

```diff
@@ -1,69 +1,69 @@
 @php
     use Filament\Tables\Actions\Position as ActionsPosition;
-    use Filament\Tables\Filters\Layout as FiltersLayout;
```

Comment:
Needs investigating

Reply:
Still there, just been moved down since it's ordered alphabetically.